baseplate/chronos.py

```diff
@@ -0,0 +1,22 @@
+"""Extensions to the standard library `datetime` module."""
+
+from datetime import datetime
+
+import pytz
```

Comment:
rather than using `pytz` and making it a hard requirement for downstream services, how about using `datetime.timezone.utc` for the UTC object? since we've dropped support for py2 that should be safe and a quick check shows they should be intercompatible if services are manually using pytz.